.jules/palette.md

```
@@ -0,0 +1,3 @@
+## 2026-08-20 - Make scrollable overflow regions keyboard focusable
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)